### PR TITLE
[Naitei] [Ecommerce]  [Guest]Show_all_product

### PR DESCRIPTION
**Pull request by SuKay712**
## Related Tickets
- ticket redmine

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Vid
Screenshot from 2024-07-25 21-31-30
Screenshot from 2024-07-25 21-32-52

eo)

Screenshot from 2024-07-25 18-47-42

### CODE REVIEW

**Comment by SuKay712** [comment]
ready

File: app/views/products/_product.html.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<div class="collage-pic">
+  <%= link_to product, class: "collage-info" do %>
+    <% if product.image.persisted? %>
+      <%= image_tag product.image, class: "collage-pic" %>
+      <p class= "center"><%= product.name%></p>
```
Comment (ghost) [inline app/views/products/_product.html.erb]:
```suggestion
      <p class= "center"><%= product.name %></p>
```

File: app/views/products/_product.html.erb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      <p><%= t("product.price", price: product.price) %></p>
```
Comment (ghost) [inline app/views/products/_product.html.erb]:
```suggestion
      <p><%= t "product.price", price: product.price %></p>
```

File: app/views/products/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<div class="collage">
+  <%= render @products %>
```
Comment (ghost) [inline app/views/products/index.html.erb]: Em dùng partial collection nhé

File: app/controllers/products_controller.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+class ProductsController < ApplicationController
+  def index
+    @pagy, @products = pagy(Product.newest, limit: Settings.page_size)
```
Comment (ghost) [inline app/controllers/products_controller.rb]:
```suggestion
    @pagy, @products = pagy Product.newest, limit: Settings.page_size
```

**Review by ghost — COMMENTED** [review]
cmt

File: app/views/products/index.html.erb (hunk 3 above)
Comment (SuKay712) [reply app/views/products/index.html.erb]: dạ em sửa rồi ạ

**Comment by SuKay712** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**